Docs: Document the plugins preinstall feature (#96464)
grafana/grafana: docs/sources/administration/plugin-management/index.md

If you're logged in to Grafana Cloud when you add a plugin, log out and then log back in again to use the new plugin.

### Install plugins using the Grafana Docker image

> To install plugins using the Grafana Docker image, you can use the `GF_INSTALL_PLUGINS` environment variable. This variable is a comma-separated list of plugins to install.

Reviewer: ```suggestion
To install plugins using the Grafana Docker image, set the `GF_INSTALL_PLUGINS` environment variable. This variable is a comma-separated list of plugins to install.
```

### Install plugin on local Grafana

> When you are hosting Grafana on your own server, you have multiple options to install a plugins. Each installation method has its own advantages and disadvantages. The different installation methods are:

Reviewer: ```suggestion
When hosting Grafana on your own server, there are multiple options to install a plugin. Each installation method has its own advantages and disadvantages. The different installation methods are:
```

> #### Install a plugin using Grafana Catalog

Reviewer: ```suggestion
#### Install a plugin using the Grafana plugin catalog
```

> In your instance, navigate to **Administration** -> **Plugins** and click in the plugin that you want to install. You can also search for the plugin using the "Search" box. Once in the plugin page, click the **Install** button or if you want to install an older version of the plugin, click the **Version history** tab and select the version that you want to install.

Reviewer: ```suggestion
In your instance, go to **Administration** -> **Plugins** and click the plugin that you want to install. 

You can also search for the plugin using the **Search** box. On the plugin page, click the **Install** button. Or, if you want to install an older version of the plugin, click the **Version history** tab and select the version that you want to install.
```

This is the easiest way to install a plugin, and it's only available to users with plugin administrator permissions.

#### Install a plugin using Grafana CLI

To install a plugin using the Grafana CLI, run the following command:

```bash
grafana cli plugins install <plugin id> <version (optional)>
```

> Similar to the previous, this method will download the plugin from the public Grafana catalog and install it in your Grafana instance.

Reviewer: ```suggestion
Similar to the previous method, this way downloads the plugin from the public Grafana catalog and installs it in your Grafana instance.
```

Use this method to install plugins if you don't have access to the Grafana UI or if you want to automate the installation process.

For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).

> #### Install a plugin using a zip file

Reviewer: ```suggestion
#### Install a plugin using a Zip file
```
